docs/getting_started.md

```diff
@@ -95,6 +99,25 @@ spec:
 
 Both labels and annotations can be added to the Prometheus configuration, so that metrics are associated with the appropriate information, such as the test and tenant ids. You can read more about this in the [Prometheus documentation](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#kubernetes_sd_config) site.
 
+##### Image pull policy and secrets
+
+You can use a private image for master and worker images, and configure its pull policy and corresponding secret. For example:
+
+```yaml title="locusttest-cr.yaml"
+apiVersion: locust.io/v1
+...
+spec:
+  image: ghcr.io/mycompany/locust:latest #(1)!
+  imagePullPolicy: Always #(2)!
+  imagePullSecrets: #(3)!
+    - gcr-secret
+  ...
+```
+
+1. [Optional] Specify which Locust image to use for both master and worker containers.
+2. [Optional] Specify the pull policy to use for containers defined within master and worker containers.
```

Comment:
would be nice to also mention the supported options. This is mainly because this implementation doesn't support "Never" which is part of what K8s API support.